frontend/src/app/main/ui/ds/controls/input_with_values.cljs

```diff
@@ -0,0 +1,68 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.main.ui.ds.controls.input-with-values
+  (:require-macros
+   [app.main.style :as stl])
+  (:require
+   [app.main.ui.ds.controls.input :refer [input*]]
+   [app.util.dom :as dom]
+   [app.util.keyboard :as kbd]
+   [rumext.v2 :as mf]))
+
+(def ^:private schema:input-with-values
+  [:map
+   [:name :string]
+   [:values :string]
+   [:on-change {:optional true} fn?]])
+
+
+(mf/defc input-with-values*
+  {::mf/props :obj
+   ::mf/forward-ref true
+   ::mf/schema schema:input-with-values}
+  [{:keys [name values on-change]}]
+  (let [editing*  (mf/use-state false)
+        editing?  (deref editing*)
+        input-ref (mf/use-ref)
+        input     (mf/ref-val input-ref)
+        title     (str name ": " values)
+        on-edit
+        (mf/use-fn
+         (fn [event]
+           (dom/stop-propagation event)
+           (reset! editing* true)
+           (dom/focus! input)))
+        on-blur
+        (mf/use-fn
+         (fn [event]
```

Comment:
Missing `on-change` in the dependencies.

Reply:
fixed

Reply:
(I also change the function name)